proto/attestation/verification.proto

```diff
@@ -146,7 +146,7 @@ message KernelLayerData {
   RawDigest kernel_cmd_line = 2 [deprecated = true];
 
   // Command-line that was passed to the kernel during startup.
```

Comment:
Consider adding a comment here stating what the significance is between an absent field and a present-but-empty field.

Reply:
+1 in fact I'm not sure I am reading the logic correctly, could you clarify the intention?

Reply:
Added a comment, more context for this change in b/325979696